presto-pinot-toolkit/pom.xml

```diff
@@ -0,0 +1,189 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.facebook.presto</groupId>
+        <artifactId>presto-root</artifactId>
+        <version>0.229-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>presto-pinot-toolkit</artifactId>
+    <name>presto-pinot-toolkit</name>
+    <description>Presto - Pinot Toolkit library</description>
+    <packaging>jar</packaging>
+
+    <properties>
+        <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.facebook.presto.pinot</groupId>
+            <artifactId>pinot-driver</artifactId>
+            <version>0.1.1</version>
```

Comment:
Let's move this to the top-level pom.xml and reference it here without the version number. An example is `<artifactId>cassandra-driver</artifactId>` dependency.